### DISCUSSION TITLE
Bad Number Format Message ??

### DISCUSSION THREAD

**Discussion by EddieHawk44** [arkypita/LaserGRBL#1974] [Q&A]
I got an Atomstack A5 Pro. Laser is not turning on when I "Blink" on Laser GRBL. The code in the program says "M3 S[$30*10/100] Bad Number Format" What does this means? Any ideas how can i fix this?
![Capture](https://user-images.githubusercontent.com/120674036/207934588-94d8572b-2bf5-4299-b523-6939de856414.PNG)

**Comment by arkypita** [comment]
Need to read grb config at least one time. Open from menu grbl, grbl configuration.

**Reply by Gyom999** [reply]
Hello I have the same problem this is my Grbl configuration

![image](https://github.com/arkypita/LaserGRBL/assets/157972862/bc59e73a-69b1-4e19-b593-c41cb3b06867)

**Comment by camiloceron2006** [comment]
como se soluciona?
tambien me pasa lo mismo

**Reply by LumineAndLine** [reply]
Have you tried refreshing the grbl config? I suddenly started having issues framing yesterday and today just opened the grbl configuration and hit "refresh" in the bottom left corner and now it's working fine again.

**Reply by Gyom999** [reply]
Finally I find the solution. I right clic on "Focus" and I modify the code. I replace M3 S[$30*10/100] with M3 S30 and YES It works !!

**Comment by Gyom999** [comment]
Finally I find the solution. I right clic on "Focus"  and I modify the code. I replace M3 S[$30*10/100] with M3 S30 and YES It works !!
